src/table/utils/accessibility-utils.ts

```diff
@@ -26,9 +26,9 @@ export const setFocusOnClosetColumnAdjuster = (anchorRef: React.RefObject<HTMLDi
 export const focusHeadMenuButton = (event: React.KeyboardEvent | React.FocusEvent<HTMLDivElement>) => {
   const target = event.target as HTMLDivElement;
   target.setAttribute("tabIndex", "-1");
-  const headMenuButton = target
-    ?.closest(".sn-table-cell")
-    ?.querySelector(".sn-table-head-menu-button") as HTMLButtonElement;
+  const baseElement = target?.closest(".sn-table-cell");
+  const headMenuButton = (baseElement?.querySelector(".sn-table-head-menu-button") ||
+    baseElement?.querySelector("#nebula-table-utils-head-menu-button")) as HTMLButtonElement;
```

Comment:
good candidate for exporting this id from shared lib, will keep it for the work that i'll do in there.

Also, `.sn-table-head-menu-button` for old head cell and `#nebula-table-utils-head-menu-button` for new head cell, so based on the flag either of those two will be true to focus

Reply:
yes that should probably be exported.

But we can't use this ID in the sn-table. Since you get one of these buttons per head cell, you get multiple instances of an element with the same ID. Having unique IDs in the DOM is a a11y requirement. So could we switch that to a class instead? Not a hurry now since we have a bunch of other a11y stuff to fix

Reply:
Yes, we (probably) can use class instead, but regarding the multiple instances part, i think that is not correct maybe. 
Because we do a querySelector on `baseElement` which is the closest el with `.sn-table-cell` class name on the triggered event location 
So we cant have multiple elements with same id i guess or am i missing something?

Reply:
the code for getting the correct element here will work. The only reason we need unique IDs is because it is a11y requirement. I recently fixed this for sn-table. (Also, I just googled, it says it has to be unique on [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id))

Reply:
will scope this out to a11y PR, 
I think it will effect PVT as well

Reply:
sounds good